src/components/Avatar/docs/examples.ts

```diff
@@ -0,0 +1,19 @@
+import { AvatarSize } from '../Avatar';
+
+export const importExample = `import { Avatar } from 'wix-ui-tpa/Avatar';`;
+
+export const example = Object.keys(AvatarSize).reduce((acc, size) => {
+  return {
+    ...acc,
+    [size]: getAvatars(size),
+  };
+}, {});
+
+function getAvatars(size) {
+  return `
+  <>
```

Comment:
Lets use `<div>` instead of `<>`.
Because this shows in the documentation.

Reply:
Done;